<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimMain</class>
 <widget class="QMainWindow" name="SimMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimMain</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addrBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Addr:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Val:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="valBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="setRAMButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startListeningButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start listening</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>398</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
